src/task_types/resmoke_tasks.rs

```diff
@@ -1187,4 +1251,18 @@ mod tests {
         assert_eq!(get_evg_fn_name(&commands[4]), Some("do multiversion setup"));
         assert_eq!(get_evg_fn_name(&commands[5]), Some("run test"));
     }
+
+    // percent_of_tests tests.
+    #[rstest]
+    #[case(100, 0.8, 80)]
+    #[case(100, 1.0, 100)]
```

Comment:
I'm not sure if we should test this, but can we add some tests to see that results are rounded correctly?
e.g. `case(101, 0.8, ??)` or `case(99, 0.8, ??)`